lib/network/modules/components/physics/CentralGravitySolver.js

```diff
@@ -35,7 +54,44 @@ class CentralGravitySolver {
     forces[node.id].x = dx * gravityForce;
     forces[node.id].y = dy * gravityForce;
   }
+
+  /**
+   * Calculate the forces the group center puts on the nodes based on the distance.
+   * @private
+   */
+  _calculateGroupForces(group, forces, node) {
+    if (group != null) {
+      let dx = group.centerX - node.x;
```

Comment:
centerX and centerY are not options that can be set, they should be calculated by the solver